modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -22,12 +22,12 @@ Ownership of the endpoints (receive or send) is automatically assigned based on
 * *Endpoints: Send to partners*
 * *Endpoints: Target at <host-name>*
 * *Endpoints: Source at <host-name>*
-. <<configure-endpoint,Configure the endpoint>> according the type of endpoint you are creating and the protocol you selected in the *New endpoint: <endpoint-type>* window.
+. <<configure-endpoint,Configure the endpoint>> according to the type of endpoint you are creating and the protocol you selected in the *New endpoint: <endpoint-type>* window.
```

Comment:
"configure the endpoint according to the type of endpoint you are creating" sounds odd and unnecessary. 
How about something like...:
```suggestion
. <<configure-endpoint,Configure the endpoint>> to be of the type you want and to be compatible with the protocol you selected in the *New endpoint: <endpoint-type>* window.
```
OR
```suggestion
. <<configure-endpoint,Configure the endpoint>> type and compatibility with the protocol you selected in the *New endpoint: <endpoint-type>* window.
```